peekingduck/cli.py

```diff
@@ -124,13 +124,17 @@ def run(
 
     curr_dir = _get_cwd()
     if config_path is None:
```

Comment:
Do we want to change the CLI option from `config_path` to `pipeline_path` too?

Reply:
To be resolved.